recipe/meta.yaml

```diff
@@ -31,17 +31,25 @@ test:
   requires:
     - pip
```

Comment:
here, please cross check bg-mpl-stylesheet as well.

Reply:
no bg-mpl-stylesheets in the conda.txt. idk if we want to add it?